ReactNative/ReactTwitter/jestSupport/Linking.js

```diff
@@ -0,0 +1,8 @@
+/* global jest */
+
+var Linking = {
+  addEventListener: jest.genMockFunction(),
+  removeEventListener: jest.genMockFunction()
+}
+
+module.exports = Linking
```

Comment:
If this is a mock you should append something like `.mock` or `-mock` in the end of the file name.